sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/AppContext.kt

```diff
@@ -155,6 +170,23 @@ object AppContext {
     private fun storeAll(networkSponsorJson: String, networkSpeakerJson: String, networkSessionJson: String) {
         dbHelper.primeAll(networkSpeakerJson, networkSessionJson, networkSponsorJson)
     }
+
+    private fun createNotificationsForSessions() = coroutineScope.lateValue.launch {
+        val mySessions = sessionQueries.mySessions().executeAsList()
```

Comment:
Should this query also be in the background?

Reply:
Yep, adding that in next commit.